pkg/tnf/operator/starter.go

```diff
@@ -46,11 +48,60 @@ var (
 	fencingUpdateMutex sync.Mutex
 )
 
+type DualReplicaClusterHandler struct {
+	ctx           context.Context
+	kubeClient    kubernetes.Interface
+	clusterStatus status.ExternalEtcdClusterStatus
+}
+
+func NewDualReplicaClusterHandler(ctx context.Context,
+	opClient v1helpers.StaticPodOperatorClient,
+	kubeClient kubernetes.Interface,
+	configInformers configv1informers.SharedInformerFactory) (*DualReplicaClusterHandler, error) {
+
+	bootstrapCompleted := false
+	readyForEtcdRemoval := false
+	dualReplicaEnabled, err := ceohelpers.IsDualReplicaTopology(ctx, configInformers.Config().V1().Infrastructures().Lister())
+	if err != nil {
+		klog.Errorf("failed to determine DualReplicaTopology, aborting controller start: %v", err)
+		return nil, err
+	} else if dualReplicaEnabled {
```

Comment:
We technically don't need this else since it's implied. (The if statement returns, so we'd save an indentation level)

Reply:
we need a linter for this :D